app/src/main/java/com/duckduckgo/app/tabs/model/TabEntitiy.kt

```diff
@@ -31,6 +31,7 @@ data class TabEntity(
     @PrimaryKey var tabId: String,
     var url: String? = null,
     var title: String? = null,
+    var skipHome: Boolean = false,
```

Comment:
In the original design, we were just planning on keeping this flag in the `viewModel` and not in the DB. why did we need to store it in the DB?

(i'm absolutely cool with this approach btw, just curious why)

Reply:
That's a good question. It's because it's the db change that triggers the live data update that then results in opening a new tab.